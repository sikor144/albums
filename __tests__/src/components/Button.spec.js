import React from 'react';
import renderer from 'react-test-renderer';

import Button from '../../../src/components/Button';

describe('Button component', () => {
  it('renders component correctly', () => {
    const component = renderer.create(
      <Button />,
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
});
