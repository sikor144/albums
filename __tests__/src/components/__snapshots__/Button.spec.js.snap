// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders component correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignSelf": "stretch",
      "backgroundColor": "#fff",
      "borderColor": "#007aff",
      "borderRadius": 5,
      "borderWidth": 1,
      "flex": 1,
      "marginLeft": 5,
      "marginRight": 5,
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "alignSelf": "center",
        "color": "#007aff",
        "fontSize": 16,
        "fontWeight": "600",
        "paddingBottom": 10,
        "paddingTop": 10,
      }
    }
  />
</View>
`;
